### DISCUSSION TITLE
Youtube video showing it

### DISCUSSION THREAD

**Discussion by jimbotel** [tjko/fanpico#14] [Ideas]
Hi @tjko ,

It would be nice if you could record some videos of a working board.

Thanks!
Jaime-

**Comment by tjko** [comment]
I don't currently have good setup for filming/editing, so I've been hoping someone better equipped to do that would be willing to do demo or review, I'd gladly send them a unit to test with...

I'll probably eventually get around to do some short videos (if no-one else volunteers to make one). But next in my todo is to create small user guide and assembly guide...

**Reply by jimbotel** [reply]
Hi @tjko ,
Thanks for your reply, I understand what you mean. It was just a suggestion, I think a short "action video" would be great.
